build_runner_core/lib/src/environment/overridable_environment.dart

```diff
@@ -35,9 +44,15 @@ class OverrideableEnvironment implements BuildEnvironment {
     Future<BuildResult> Function(BuildResult result,
             FinalizedAssetsView finalizedAssetsView, AssetReader reader)
         finalizeBuild,
+    Future<BuildScriptUpdates> Function(
+    AssetReader assetReader,
```

Comment:
Is this dartformatted? We should file a bug if so that doesn't look right.

You could try removing the trailing comma here too, that might make it behave better

Reply:
Fixed by removing the trailing comma